### PR TITLE
eve/http: add request/response headers test

### PR DESCRIPTION
**Pull request by maurizioabba**
pcap downloaded from https://wiki.wireshark.org/SampleCaptures?action=AttachFile&do=get&target=http.cap

### CODE REVIEW

File: tests/http-all-headers/suricata.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+%YAML 1.1
+---
+
+include: ../../etc/suricata-4.0.3.yaml
```
Comment (victorjulien) [inline tests/http-all-headers/suricata.yaml]: shouldn't be necessary

File: tests/http-all-headers/test.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+requires:
```
Comment (victorjulien) [inline tests/http-all-headers/test.yaml]: min-version: 5.0.0

File: tests/http-all-headers/check.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#! /bin/sh
```
Comment (victorjulien) [inline tests/http-all-headers/check.sh]: @jasonish could this be expressed in the test.yaml itself instead?

**Comment by victorjulien** [comment]
Can you add a README that links to the pcap source?

**Comment by maurizioabba** [comment]
Replaced by https://github.com/OISF/suricata-verify/pull/12

**Pull request closed without merging**